src/components/formField/formField.component.tsx

```diff
@@ -0,0 +1,33 @@
+import cN from 'classnames';
+import React, { FC } from 'react';
+import { useField } from 'formik';
+import { IFormField } from '@app/components/formField/formField.interface';
+import { style as useStyle } from '@app/components/modals/movieDetailsModal/movieDetailsModal.component.style';
```

Comment:
you can't you move this style here and not reuse all movieDetailsModal styles?

Reply:
Nice catch! Looks like it was a temporary solution, and I just forgot to make it better.
Fixed.